packages/fxa-content-server/app/scripts/views/settings/avatar_crop.js

```diff
@@ -131,6 +126,14 @@ const View = FormView.extend({
   _onZoomRangeChange() {
     this.logViewEvent('zoom.range');
   },
+
+  _returnToAvatarChange() {
+    this.cropper.destroy();
+  },
+
+  _returnToSettings() {
+    this.cropper.destroy();
+  },
```

Comment:
@jodyheavener this is my bad, I think this should be the last change before this sucker finally gets merged. `destroy()` above didn't work because I _meant_ to use `remove()`. 🤦‍♀️

You should be able to remove `_returnToAvatarChange()`, `_returnToSettings()`, and `this.cropper.destroy();` on line 96 and do this instead:

```
  remove() {
    this.cropper.destroy();
  }
```

I just tested it and it looks like it's working for me.

Reply:
You are correct. I've updated accordingly!